### PR TITLE
Toggle answer

### PR DESCRIPTION
**Pull request by marcbendixen**
Add function to toggle answer on quiz card.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,13 +6,60 @@
...
+          role="button"
+          aria-label="Toggle bookmark"
+        >
+          <svg
+            aria-hidden="true"
+            focusable="false"
+            data-prefix="fas"
+            data-icon="bookmark"
+            class="svg-inline--fa fa-bookmark fa-w-12"
+            role="img"
```
Comment (macfrei) [inline src/index.html:32]: According to MDN you should also set an `aria-label` when you use `role="img"` on an SVG
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img#svg_and_roleimg

File: src/index.html [hunk 2]
```diff
@@ -99,10 +320,52 @@ <h2 class="quiz-card__heading">Question</h2>
         </ul>
       </section>
     </main>
-
+    <!-- create -->
+    <main class="page-create hidden">
+      <form method="post" class="create-form">
+        <div class="create-form__input-group">
+          <label for="question" class="create-form__heading">Question</label>
```
Comment (macfrei) [inline src/index.html:327]: This is not a heading but a label
```suggestion
          <label for="question" class="create-form__label">Question</label>
```

File: src/index.html [hunk 3]
```diff
@@ -99,10 +320,52 @@ <h2 class="quiz-card__heading">Question</h2>
...
+          <label for="answer" class="create-form__heading">Answer</label>
+          <textarea
+            name="answer"
+            id="answer"
+            cols="30"
+            rows="8"
+            placeholder="Add your answer here ..."
+            required
+          ></textarea>
+        </div>
```
Comment (macfrei) [inline src/index.html:347]: Another way also, to save yourself a `div` element 😏 You could nest the textarea within the label and then give the styling you give the div to the label. Should work the same :)
```suggestion
       <label class="create-form__label">Answer
          <textarea
            name="answer"
            cols="30"
            rows="8"
            placeholder="Add your answer here ..."
            required
          ></textarea>
        </label>
```

File: src/index.html [hunk 4]
```diff
@@ -99,10 +320,52 @@ <h2 class="quiz-card__heading">Question</h2>
...
+        <button class="button" type="submit">Submit</button>
+      </form>
+    </main>
+    <!-- profile -->
+    <main class="page-profile hidden">Profile</main>
     <footer>
       <nav class="navigation">
-        <a href="/" class="navigation__item navigation__item--active">
+        <a
```
Comment (macfrei) [inline src/index.html:365]: You should change these to `button` elements. An anchor element is referencing another page or sub-page.

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  headingHome.classList.add('hidden')
+  headingBookmarks.classList.add('hidden')
+  headingCreate.classList.add('hidden')
+  headingProfile.classList.remove('hidden')
+})
+
+// toggle bookmark icon
+const bookmarkIcons = document.querySelectorAll('.quiz-card__bookmark-icon')
+
+bookmarkIcons.forEach(icon => {
```
Comment (macfrei) [inline src/index.js:89]: Sneaky little `forEach` 😎

**Review by macfrei — COMMENTED** [review]
Well done, added a few thoughts! 🚀

File: src/index.js (hunk 5 above)
Comment (marcbendixen) [reply src/index.js:89]: I couldn't resist ;-)

**Pull request merged**